tests/unit/test_context.py

```diff
@@ -72,84 +88,91 @@ def test_explain():
     )
 
 
-def test_sql():
+@pytest.mark.parametrize("gpu", [False, pytest.param(True, marks=pytest.mark.gpu)])
+def test_sql(gpu):
```

Comment:
It looks like `create_table(gpu=True)` doesn't result in a dask-cudf dataframe right now, so we'll need to xfail this until that's resolved.